cluster-scope/overlays/ocp-prod/externalsecrets/clientsecret-moc-testing.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "kubernetes-client.io/v1"
+kind: ExternalSecret
+metadata:
+  name: clientsecret-moc-testing
+  namespace: openshift-config
+spec:
+  backendType: secretsManager
+  data:
+    - key: cluster/ocp-prod/openshift-config/clientsecret-moc-testing
+      name: clientSecret
```

Comment:
It would be nice to have the word `sso` somewhere in the secrets. Right now it's just some client secret.

Reply:
I guess it would be in the secret name so that the rest of the key matches what's in openshift.

Now that I think of it, could we actually have some other "client secret" in the same openshift-config namespace that makes it necessary to keep this name distinct.

Reply:
I've updated the PR to name things `sso-clientsecret...` instead of just `clientsecret...`, which I think addresses your concern.

> Now that I think of it, could we actually have some other "client secret" in the same openshift-config namespace that makes it necessary to keep this name distinct?

I don't think so, but it also doesn't hurt to just rename it as you originally requested :)